src/web/nextui/src/app/eval/ResultsView.tsx

```diff
@@ -212,6 +221,17 @@ export default function ResultsView({
     [table.body],
   );
 
+  const promptOptions = useMemo(() => {
```

Comment:
nit: we use `React.useMemo`, `React.useEffect` in most places elsewhere in the codebase. Let's prefer to keep the `React` namespace